components/browser/awesomebar/src/main/java/mozilla/components/browser/awesomebar/BrowserAwesomeBar.kt

```diff
@@ -50,6 +50,7 @@ class BrowserAwesomeBar @JvmOverloads constructor(
     internal var listener: (() -> Unit)? = null
     internal var editSuggestionListener: ((String) -> Unit)? = null
     internal val styling: BrowserAwesomeBarStyling
+    var shouldRotateEditArrow: Boolean = false
```

Comment:
This API is overly specific. Maybe instead we can introduce some top/bottom mode that can control this and potentially other things in the future.

Reply:
I have changed the naming to `usingBottomBrowserToolbar`. I think a Boolean should be enough because we have only two possible positions for the toolbar.